src/components/rating-stars/rating-stars.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Star from "../star/star";
+import {withOneActiveElement} from "../../hocs/withOneActiveElement";
+
+const STARS_COUNT = 5;
+
+const RatingStars = (props) => {
+  const {
+    activeElementIndex,
+    handleActiveChange,
+    handleFieldDataChange
+  } = props;
+
+  return (
+    <div className="rating">
+      <div className="rating__stars">
+        {new Array(STARS_COUNT).fill(``).map((value, index) =>
```

Comment:
Можно скипнуть аргумент fill. И если value не используется, лучше это пометить
```suggestion
        {new Array(STARS_COUNT).fill().map((_, index) =>
```

Reply:
С нижним подчёркианием крутой ход, запомню!